UPGRADE.md

```diff
@@ -12,8 +12,8 @@ Follow instructions in a [monorepo upgrade guide](docs/contributing/upgrading-mo
 * upgrade only your composer dependencies and follow instructions
 * check all instructions in all sections, any of them could be relevant for you
 * if you want update your project with the changes from [shopsys/project-base],
-    you can follow the *(optional)* instructions or cherry-pick from the repository whatever is relevant for you but we do not recommend rebasing or merging everything because the changes might not be compatible with your project as it probably evolves in time
-* instructions marked as *(optional)* are not vital, but could be helpful,
+    you can follow the *(low priority)* instructions or cherry-pick from the repository whatever is relevant for you but we do not recommend rebasing or merging everything because the changes might not be compatible with your project as it probably evolves in time
+* instructions marked as *(low priority)* are not vital, but could be helpful,
```

Comment:
* instructions marked as *(low priority)* are not vital and can be done last,
  and we recommend to perform them as well during upgrading as it might ease your work in the future

or something like that. To be clear, we really recommending doing them.